### PR TITLE
 Move to Rust 2018 edition

### PR DESCRIPTION
**Pull request by dbrgn**
After the preparation done in #83, not much work was left to be done.

Based on #87, that needs to be merged first.

### CODE REVIEW

**Comment by dbrgn** [comment]
@rnestler ready for review now :slightly_smiling_face:

**Comment by rnestler** [comment]
@dbrgn ping.

File: src/lib.rs [hunk 1]
```diff
@@ -6,16 +6,8 @@
 #![doc(html_root_url = "https://docs.rs/spaceapi-server")]
 
 #[macro_use] extern crate log;
```
[the remark this answers is not in the document]
Comment (dbrgn) [reply src/lib.rs]: fixed!

**Comment by g5pw** [comment]
awww i rebased it but it looks like I don't have access to push :/ I'm in the spaceapi-community group already, though :/

**Comment by rnestler** [comment]
> awww i rebased it but it looks like I don't have access to push :/ I'm in the spaceapi-community group already, though :/

I think that's because it is a branch @dbrgn created? You could just push it to a new branch then.

**Comment by dbrgn** [comment]
Rebased!

**Comment by dbrgn** [comment]
@g5pw finally :slightly_smiling_face:

**Pull request merged**